plugins/catalog-react/src/components/InspectEntityDialog/components/AncestryPage.tsx

```diff
@@ -213,13 +213,11 @@ export function AncestryPage(props: { entity: Entity }) {
         {t('inspectEntityDialog.ancestryPage.title')}
       </DialogContentText>
       <DialogContentText gutterBottom>
-        This is the ancestry of entities above the current one - as in, the
-        chain(s) of entities down to the current one, where{' '}
+        {t('inspectEntityDialog.ancestryPage.descriptionPart1')}{' '}
         <Link to="https://backstage.io/docs/features/software-catalog/life-of-an-entity">
-          processors emitted
+          {t('inspectEntityDialog.ancestryPage.processorsLink')}
         </Link>{' '}
-        child entities that ultimately led to the current one existing. Note
-        that this is a completely different mechanism from relations.
+        {t('inspectEntityDialog.ancestryPage.descriptionPart2')}
```

Comment:
Hey @Eswaraiahsapram, this way, this string is really hard to translate later. For example keeping spaces might not make sense in some languages.

The outer "description" should be at least one translation string that gets the link passed as a prop.

See https://backstage.io/docs/plugins/internationalization/ "use within a component" example